usaspending_api/download/sql/account_balances.sql

```diff
@@ -0,0 +1,12 @@
+DROP VIEW IF EXISTS vw_account_balances;
+CREATE VIEW vw_account_balances AS
+SELECT
+    "appropriation_account_balances".*,
```

Comment:
I'm sure you probably copy-pasted from django output for getting this up quickly, but it'd be clean and nice to get rid of the double quotes. Optional to alias the tables if you'd like. This and the other views.

Reply:
Oh 100% a copy-paste and cleanup. Will get this cleaned up.